functions.php

```diff
@@ -2,7 +2,7 @@
 
 function child_enqueue_styles() {
 
-	wp_enqueue_style( 'ast-child-style', get_stylesheet_directory_uri() . '/style.css', array('astra-theme-css'), AST_THEME_VERSION, 'all' );
+	wp_enqueue_style( 'astra-child-style', get_stylesheet_directory_uri() . '/style.css', array('astra-theme-css'), ASTRA_THEME_VERSION, 'all' );
```

Comment:
We should use different version number for the child theme, create a different constant for the child theme and use it in here in place of `ASTRA_THEME_VERSION `